streamx-console/streamx-console-service/src/assembly/plugins/Dockerfile

```diff
@@ -15,7 +15,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 ###############################################################################
-FROM maven:3.8.5-jdk-8
+FROM docker:dind
```

Comment:
can we use build cache?

Reply:
@czy006 It is necessary to use docker:dind because streamx running on top of k8s needs to be docker capable, which faces the docker in docker problem. docker:dind is one of the solutions to this problem